appeals/web/src/server/appeals/appeal-details/interested-party-comments/interested-party-comments.service.js

```diff
@@ -0,0 +1,27 @@
+import { paginationDefaultSettings } from '#appeals/appeal.constants.js';
+
+/**
+ * Fetch paginated appeal comments based on appeal ID and status.
+ *
+ * @param {import('got').Got} apiClient
+ * @param {string} appealId
+ * @param {string|undefined} statusFilter
+ * @param {number} pageNumber
+ * @param {number} pageSize
+ * @returns {Promise<any>}
+ */
+export const getInterestedPartyComments = (
```

Comment:
I took this from https://github.com/Planning-Inspectorate/appeals-back-office/pull/437 to try and minimise conflicts